Documentation/JobManager/README.md

```diff
@@ -7,35 +7,34 @@ The Job Manager is a toolkit component that you can just plug into the applicati
 ### Usage
 
 ```swift
+// create and load a task
+let task = AGSGeodatabaseSyncTask(url: URL)
+task.load{ error in
 
-	// create and load a task
-        let task = AGSGeodatabaseSyncTask(url: URL)
-	task.load{ error in
+    // make sure we are still around...
+    guard let strongSelf = self else {
+	return
+    }
 
-            // make sure we are still around...
-            guard let strongSelf = self else {
-                return
-            }
+    // make sure task does not get released
+    guard let strongTask = task else{
+	return
```

Comment:
The indentation for this return statement looks off. Same with the guard statement above.

Reply:
Good catch. Looks like there were tabs instead of spaces, and the wrong amount